.github/workflows/simulation.yml

```diff
@@ -236,7 +234,7 @@ jobs:
         make -f src/posix/Makefile-posix
     - name: Run
       run: |
-        OT_NCP_PATH="$(pwd)/$(ls output/posix/*/bin/ot-ncp)" RADIO_DEVICE="$(pwd)/$(ls output/*/bin/ot-rcp)" make -f src/posix/Makefile-posix check
+        VERBOSE=1 OT_NCP_PATH="$(pwd)/$(ls output/posix/*/bin/ot-ncp)" RADIO_DEVICE="$(pwd)/$(ls output/*/bin/ot-rcp)" make -f src/posix/Makefile-posix check
```

Comment:
`VERBOSE` in env not working?

Reply:
It is only used by the `make check`, so I prefer adding it here.